### DISCUSSION TITLE
Is there a Capacitor equivalent to the preference tag from Cordova?

### DISCUSSION THREAD

**Discussion by johnborges** [ionic-team/capacitor#4355] [Q&A]
I'm trying to find a way to set custom config values at the app level that are accessible by Capacitor plugins. In Cordova, we previously had `<preference />`.

**Comment by jcesarmobile** [comment]
Cordova plugins? Capacitor plugins?

**Reply by johnborges** [reply]
Capacitor plugins.

**Reply by jcesarmobile** [reply]
You can put a "plugins" object in `capacitor.config.json` file and inside it another object with the plugin name.
Check Splash Screen plugin:

https://capacitorjs.com/docs/apis/splash-screen#hiding-the-splash-screen

To get the value
android:
https://github.com/ionic-team/capacitor-plugins/blob/main/splash-screen/android/src/main/java/com/capacitorjs/plugins/splashscreen/SplashScreenPlugin.java#L76
ios:
https://github.com/ionic-team/capacitor-plugins/blob/main/splash-screen/ios/Plugin/SplashScreenPlugin.swift#L61

**Reply by johnborges** [reply]
Yes, but what about a use case where multiple plugins need to reuse a single config value? Something not scoped to a particular plugin. My use case is having a number of Capacitor plugins each needing to reference something like `getConfigValue("backgroundColor")`

**Reply by jcesarmobile** [reply]
Then just put any value on the root of the `capacitor.config.json` and read it with the config object

**Reply by johnborges** [reply]
Got it. I can use `self.bridge?.config.getValue`

**Comment by Howard-Lam-UnitedVanning** [comment]
How do I read config for web plugin?
